dts/msm8974/rules.mk

```diff
@@ -4,4 +4,5 @@ DTBS += \
 	$(LOCAL_DIR)/msm8974-lge-d855.dtb \
 	$(LOCAL_DIR)/msm8974-lge-hammerhead.dtb \
 	$(LOCAL_DIR)/msm8974-samsung-klte.dtb \
+	$(LOCAL_DIR)/msm8974-samsung-kltechn.dtb \
```

Comment:
Those should've been probably named "msm8974-samsung-r14" and "-r10" but those surely should be consistent so maybe fine for now...

Reply:
If so, I think we could probably add all variants of S5 according to the device tree and just mark them as `Untested` in the README.

Reply:
I agree with r10/r14 now, considering only this is distinguishable via msm-id.

Reply:
I prefer `-r10`/`-r14` too. So the problem is: is changing the filename of `msm8974-samsung-klte.dtb` acceptable? If so, I will do that.

Reply:
Yes, I'd appreciate a dedicated commit renaming that file.